<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C5SettingsWidget</class>
 <widget class="QWidget" name="C5SettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>429</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Waiter</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="2">
       <widget class="C5LineEdit" name="leServiceFactor">
        <property name="Tag" stdset="0">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Service value</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Receipt printer</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="2">
       <widget class="C5LineEdit" name="leIDRAMID">
        <property name="Tag" stdset="0">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="C5LineEdit" name="leIDRAMPhone">
        <property name="Tag" stdset="0">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Tax department</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>IDRAM Id</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>IDRAM phone</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Tax IP</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="C5LineEdit" name="leTaxPort">
        <property name="Tag" stdset="0">
         <number>6</number>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="C5LineEdit" name="leTaxIP">
        <property name="Tag" stdset="0">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="C5LineEdit" name="leTaxPassword">
        <property name="Tag" stdset="0">
         <number>7</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="C5LineEdit" name="leLocalReceiptPrinter">
        <property name="Tag" stdset="0">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Tax port</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Tax password</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="C5LineEdit" name="leTaxDept">
        <property name="Tag" stdset="0">
         <number>8</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>C5LineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">c5lineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leLocalReceiptPrinter</tabstop>
  <tabstop>leServiceFactor</tabstop>
  <tabstop>leIDRAMID</tabstop>
  <tabstop>leIDRAMPhone</tabstop>
  <tabstop>leTaxIP</tabstop>
  <tabstop>leTaxPort</tabstop>
  <tabstop>leTaxPassword</tabstop>
  <tabstop>leTaxDept</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
